ocaml-lsp-server/src/hover_req.ml

```diff
@@ -37,8 +42,35 @@ let handle server { HoverParams.textDocument = { uri }; position; _ } =
       match Document.kind doc with
       | `Other -> Fiber.return None
       | `Merlin merlin -> (
+        let verbosity =
+          let mode =
+            match (mode, Sys.getenv_opt "OCAMLLSP_HOVER_IS_EXTENDED") with
```

Comment:
You might want to hoist up getting the environment variable to the top so it's only done once.